compiler/luci/service/src/ShapeInfer_StridedSlice.cpp

```diff
@@ -301,7 +301,7 @@ StridedSliceParams BuildStridedSliceParams(StridedSliceContext *op_context)
     if ((1 << i) & effective_ellipsis_mask)
     {
       // If ellipsis_mask, set the begin_mask and end_mask at that index.
-      added_ellipsis = std::max(0l, i - ellipsis_start_idx);
+      added_ellipsis = std::max(int64_t(0), i - ellipsis_start_idx);
```

Comment:
Out of curiosity, `ULL` may work. I will try.
```suggestion
      added_ellipsis = std::max(0ULL, i - ellipsis_start_idx);
```
Your solution is also okay. : )

Reply:
```
[   59s] /home/abuild/rpmbuild/BUILD/nnfw-1.25.0/compiler/luci/service/src/ShapeInfer_StridedSlice.cpp: In function '{anonymous}::StridedSliceParams {anonymous}::BuildStridedSliceParams({anonymous}::StridedSliceContext*)':
[   59s] /home/abuild/rpmbuild/BUILD/nnfw-1.25.0/compiler/luci/service/src/ShapeInfer_StridedSlice.cpp:304:61: error: no matching function for call to 'max(long long unsigned int, int64_t)'
[   59s]   304 |       added_ellipsis = std::max(0ULL, i - ellipsis_start_idx);
```
`0ULL` has same issue because `i - ellipsis_start_idx` is not `uint64_t`. Maybe you mean `0LL`?

Reply:
I prefer to use cstdint type.

Reply:
Personally I prefer `LL` because it is also a part of standard and is short and succinct, and others seem to use `ll` or `LL` for 64bit integer literals. Anyway I honor your preference.